SEBrowser/Scripts/TSX/global.d.ts

```diff
@@ -93,7 +93,7 @@ export namespace SEBrowser {
     interface IReportTimeFilter { date: string, time: string, windowSize: number, timeWindowUnits: number }
     interface IEventCharacteristicFilters {
         durationMin: number, durationMax: number,
-        Phase: { A: boolean, B: boolean, C: boolean },
+        Phase: { AN: boolean, BN: boolean, CN: boolean, AB: boolean, BC: boolean, ABG: boolean, BCG: boolean, ABC: boolean, ABCG: boolean }
```

Comment:
move the Phases into a new interface `IPhaseFilters` to make this cleaner and easier to read